examples/view_multi_25d.html

```diff
@@ -29,18 +29,37 @@
             var parent;
             var index;
             var wms;
+            var wmsUrl;
             var obj;
             var offset;
             var tileLayer;
             var config;
 
-            var wmsLayers = [
-                'fpc_fond_plan_communaut.fpcilot',
-                'pvo_patrimoine_voirie.pvochausseetrottoir',
-                'Ortho2009_vue_ensemble_16cm_CC46',
-                'pos_opposable.poshauvoi',
-                'MNT2015_Ombrage_2m',
-                'cad_cadastre.cadilot',
+            var wmsSources = [
+                {
+                    name: 'fpc_fond_plan_communaut.fpcilot',
+                    url: 'https://download.data.grandlyon.com/wms/grandlyon',
+                },
+                {
+                    name: 'pvo_patrimoine_voirie.pvochausseetrottoir',
+                    url: 'https://download.data.grandlyon.com/wms/grandlyon',
+                },
+                {
+                    name: 'ortho_latest',
+                    url: 'https://imagerie.data.grandlyon.com/wms/grandlyon',
+                },
+                {
+                    name: 'pos_opposable.poshauvoi',
+                    url: 'https://download.data.grandlyon.com/wms/grandlyon',
+                },
+                {
+                    name: 'MNT2015_Ombrage_2m',
+                    url: 'https://download.data.grandlyon.com/wms/grandlyon',
+                },
+                {
+                    name: 'cad_cadastre.cadilot',
+                    url: 'https://download.data.grandlyon.com/wms/grandlyon',
+                },
```

Comment:
Except for the `ortho_latest` layer, I did not manage to find equivalent in the imagerie.data.grandlyon.com [GetCapabilites](https://imagerie.data.grandlyon.com/wms/grandlyon?SERVICE=wms&REQUEST=GetCapabilities) for the other layers.

Reply:
Maybe using this url : 'https://data.grandlyon.com/geoserver/metropole-de-lyon/ows'

Looks like it works, hopefully it's the updated one